CONTRIBUTING.md

```diff
@@ -4,22 +4,3 @@ We welcome contributions to Serac. To do so please submit a pull request through
 Serac github page at https://github.com/LLNL/serac.
 
 All contributions to Serac must be made under the BSD License.
-
-Any questions can be sent to bramwell1@llnl.gov.
-
-# Attribution
-
-The Serac project uses git's commit history to track contributions from individual developers. 
```

Comment:
Thank you! I have at times contemplated about updating this. Maybe leave these three lines